src/Perpetuum/Services/Daytime/DayTime.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace Perpetuum.Services.Daytime
+{
+    public static class DayTime
```

Comment:
It may be a bit pedantic, but maybe we can call i `TimeOfDay` instead of `DayTime`?

Reply:
I think in this instance `GameDay` or `GameClock` is more extensible as a class title